backend/drive/drive.go

```diff
@@ -3969,16 +3994,16 @@ func (f *Fs) Command(ctx context.Context, name string, arg []string, opt map[str
 			dir = arg[0]
 		}
 		return f.unTrashDir(ctx, dir, true)
-	case "copyid":
-		if len(arg)%2 != 0 {
-			return nil, errors.New("need an even number of arguments")
+	case "copyid", "moveid":
+		if len(arg)%2 == 0 {
+			return nil, errors.New("need an odd number of arguments")
 		}
 		for len(arg) > 0 {
 			id, dest := arg[0], arg[1]
-			arg = arg[2:]
-			err = f.copyID(ctx, id, dest)
+			arg = arg[3:]
```

Comment:
This should say `arg = arg[2:]` to pop the used arguments off for the `for` loop